components/function-controller/internal/webhook/defaulting_webhook_test.go

```diff
@@ -86,6 +86,7 @@ func TestDefaultingWebHook_Handle(t *testing.T) {
 			},
 			args: args{
 				ctx: context.Background(),
+				//nolint
 				req: admission.Request{
```

Comment:
We can add the field name, because it will take longer investigating why `nolint` was added.